packages/ckeditor5-dev-tests/tests/utils/automated-tests/getkarmaconfig.js

```diff
@@ -71,4 +71,57 @@ describe( 'getKarmaConfig', () => {
 		expect( karmaConfig ).to.have.own.property( 'browsers' );
 		expect( karmaConfig ).to.have.own.property( 'singleRun', true );
 	} );
+
+	describe( 'BrowserStack integration', () => {
+		beforeEach( () => {
```

Comment:
What about tear down after overriding `process.env`? This is a global object so it's better to use stubbing and sandboxes.

Reply:
It's already done https://github.com/ckeditor/ckeditor5-dev/blob/19184b1fd67f50c904cd2acc825cfbeb3a37d45c/packages/ckeditor5-dev-tests/tests/utils/automated-tests/getkarmaconfig.js#L23-L42.